binding/client.go

```diff
@@ -368,7 +369,7 @@ func (r *Client) BucketPut(source, destination string) (err error) {
 			if isDir {
 				err = r.putDir(part, source)
 			} else {
-				err = r.putFile(part, source)
+				err = r.loadFile(part, source)
```

Comment:
Understood, but this change came in time when there was another function calling the loadFile, which is gone since you added FilePost method (which is a better solution for this). So the putFile _is_ called only from BucketPut, so I'd would not recommend changing it (as actually unrelated code to this test/PR).